.github/workflows/create-release.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           token: ${{ secrets.GH_TOKEN }}
           release-type: simple
           package-name: dsp-api
-          pull-request-title-pattern: "chore${{ scope }}: Release ${{ version }}"
+          pull-request-title-pattern: "chore${scope}: Release${component} ${version}."
```

Comment:
Just remove the component maybe, which I think we don't use anyway.

Reply:
Good point, done.

Reply:
Not that good as we see :D